src/Appium.Net/Appium/Interfaces/ISendsKeyEvents.cs

```diff
@@ -13,22 +13,36 @@
 //limitations under the License.
 
 
+using OpenQA.Selenium.Appium.Android;
+
 namespace OpenQA.Selenium.Appium.Interfaces
 {
     public interface ISendsKeyEvents : IExecuteMethod
     {
         /// <summary>
-        /// Sends a device key event with metastate
+        /// Sends a device key event with meta state
         /// </summary>
         /// <param name="keyCode">Code for the long key pressed on the Android device</param>
         /// <param name="metastate">metastate for the long key press</param>
         void PressKeyCode(int keyCode, int metastate = -1);
```

Comment:
I'm not quite sure if -1 is a valid default value. in binary representation this could look like 0xFFFFFFFF, which enables all flags. I would rather set it to 0